src/components/application_manager/src/commands/hmi/request_to_hmi.cc

```diff
@@ -56,6 +56,21 @@ bool ChangeInterfaceState(ApplicationManager& application_manager,
                                                            interface_state);
     return is_available;
   }
+  if (response_from_hmi[strings::params].keyExists(hmi_response::code)) {
+    int32_t response_code_ =
```

Comment:
@ValeriiMalkov why local variable with underscore?

Reply:
@LuxoftAKutsan That was fixed and force updated. See commit 423b389c93a5625b3bd4b27c90b68dc134877e7b